pkg/operations/start.go

```diff
@@ -20,8 +21,14 @@ import (
 )
 
 func StartVM(vm *vmmd.VM, debug bool) error {
+	// Get the Docker client
+	dc, err := docker.GetDockerClient()
```

Comment:
make all these calls use the global provider

Reply:
:+1: